libcaf_net/caf/net/datagram_transport.cpp

```diff
@@ -0,0 +1,97 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#include "caf/net/datagram_transport.hpp"
+
+namespace {
+
+// Name of the component for logging.
+static constexpr std::string_view component = "datagram_transport";
```

Comment:
That's not quite how the components are meant to be used. The component should be the module, not the class. We already have `log::net::(debug|...)` for that purpose. The functions taking a `component` shouldn't be used directly.